bluenovaio/otto
# BN-517 | Initial CLI tool

By submitting a PR to this repository, you agree to the terms within the [Code of Conduct](/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

>  This provides the initial structure of our CLI tool where we can start to add Otto commands.


### References

> Include any links supporting this change such as a:
>
> BN-517 Jira Ticket

### Testing

1. Clone the Repo
2. Build the project
3. Run "otto example" in the command line in the root folder of the project. You should see the console output a successful message.

### Checklist
